### DISCUSSION TITLE
Enabling the option to create internal repositories

### DISCUSSION THREAD

**Discussion by fhein93** [community/community#22469] [New to GitHub]
<p>Hi everyone,</p>
<p>I am working in IT-Support and never had to work with GitHub, so please bear with me :slight_smile:</p>
<p>It became my task to enable internal repositories in our companies GitHub account. We are on the enterprise plan, which in theory enables us to create internal repositories. The issue is that I can’t find the option to create a new one or to migrate an existing repo to internal.</p>
<p>Is there an option I missed? I am an organization admin, so I should have access rights to change everything that is needed.</p>
<p>I can’t find a documentation on this topic - is there anything I need to activate?</p>
<p>This is the blog entry about the feature: <a href="https://github.blog/2019-11-05-internal-repositories-are-now-generally-available-for-github-enterprise/" class="inline-onebox" rel="noopener nofollow ugc">Internal repositories are now generally available for GitHub Enterprise | The GitHub Blog</a></p>
<p>Thanks alot!</p>

**Comment by byrneh** [comment]
<p>Hi <a class="mention" href="/fhein93">@fhein93</a> welcome to the community. Your solution should be in the links below<br>
<a href="https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/restricting-repository-creation-in-your-organization" rel="noopener nofollow ugc">restricting-repository-creation-in-your-organization</a></p>
<p>You may also be interested in this setting <a href="https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/restricting-repository-visibility-changes-in-your-organization" rel="noopener nofollow ugc">restricting-repository-visibility-changes-in-your-organization</a></p>
<p>It is also possible to enforce policies to organization and repositories from you enterprise policy, and example here <a href="https://docs.github.com/en/enterprise-cloud@latest/admin/policies/enforcing-policies-for-your-enterprise/enforcing-repository-management-policies-in-your-enterprise#enforcing-a-policy-for-repository-creation" rel="noopener nofollow ugc">enforcing-repository-management-policies-in-your-enterprise#enforcing-a-policy-for-repository-creation</a></p>

**Comment by fhein93** [comment]
<p>Hi <a class="mention" href="/byrneh">@byrneh</a>,</p>
<p>thanks for answering. I do not have the option to select internal in the guides that you provided.<br>
I made sure we have the correct plan (which is enterprise).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/2/62e53d54a9dee0a7ed319e72d03e0b3d7abddd06.png" data-download-href="/uploads/short-url/e6RW0xsCRoCuEmv3OJE6dSQkFtY.png?dl=1" title="Bildschirmfoto 2022-03-02 um 08.54.56" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/92791725/181071284-8147ab9d-8921-4a6e-abf9-48438f98cd5b.png" alt="Bildschirmfoto 2022-03-02 um 08.54.56" data-base62-sha1="e6RW0xsCRoCuEmv3OJE6dSQkFtY" width="690" height="254" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62e53d54a9dee0a7ed319e72d03e0b3d7abddd06_2_690x254.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62e53d54a9dee0a7ed319e72d03e0b3d7abddd06_2_1035x381.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/2/62e53d54a9dee0a7ed319e72d03e0b3d7abddd06.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/2/62e53d54a9dee0a7ed319e72d03e0b3d7abddd06_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Bildschirmfoto 2022-03-02 um 08.54.56</span><span class="informations">1275×470 64.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by byrneh** [comment]
<p>raise an enterprise support ticket asking them to investigate</p>

**Comment by fhein93** [comment, marked as answer]
<p>I talked to GitHub support. An enterprise account is different from an enterprise organization, which is needed in order to create them.</p>
<p>Thanks alot</p>
